lib/iterable_flutter.dart

```diff
@@ -66,13 +68,40 @@ class IterableFlutter {
 
   static Future<dynamic> nativeMethodCallHandler(MethodCall methodCall) async {
     final arguments = methodCall.arguments as Map<dynamic, dynamic>;
+    final argumentsCleaned = sanitizeMap(arguments, Platform.isAndroid);
 
     switch (methodCall.method) {
       case "openedNotificationHandler":
-        _onOpenedNotification?.call(arguments);
+        _onOpenedNotification?.call(argumentsCleaned);
         return "This data from native.....";
       default:
         return "Nothing";
     }
   }
+
+  static Map<String, dynamic> sanitizeMap(
+    Map<dynamic, dynamic> mapDynamic,
```

Comment:
I'd rename this variable with a standard naming convention:
ie: `dynamicsMap` , `dynamicMap`, `schemaMap`, `toMap`

you could pick the option fits the context